sys/net/vnet.h

```diff
@@ -287,8 +287,26 @@ extern struct sx vnet_sxlock;
 #define	VNET_DEFINE_STATIC(t, n) \
     static t VNET_NAME(n) __section(VNET_SETNAME) __used
 #endif
+#ifdef __CHERI_PURE_CAPABILITY__
+/*
+ * XXX: This cannot use exact bounds currently as vnet variables are
```

Comment:
Comment block should be moved to the line below so it annotates _VNET_PTR specifically